woodwork/column_accessor.py

```diff
@@ -106,6 +106,13 @@ def _series(self):
     def schema(self):
         return copy.deepcopy(self._schema)
 
+    @property
+    @_check_column_schema
+    def nullable(self):
+        """Whether the column schema contains a nullable logical type."""
```

Comment:
This might be better as 

> Whether the column can contain null values 

Or something that focuses less on the column schema and logical types.

Reply:
Yep, I agree that's better.